### PR TITLE
Update prebuild version

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Update prebuild version, was oudated and then failed in the release workflow

## Type of change

- Bug fix (a non-breaking change which fixes an issue)

## How the change has been tested

`yarn prebuild-x64`

### CODE REVIEW

File: bindings/nodejs/CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 1.1.3 - 2023-10-27
```
Comment (thibault-martinez) [inline bindings/nodejs/CHANGELOG.md:22]: So 1.1.2 has been released bu without prebuilds ?

Comment (Thoralf-M) [reply bindings/nodejs/CHANGELOG.md:22]: Yes https://www.npmjs.com/package/@iota/sdk/v/1.1.2

Comment (Thoralf-M) [reply bindings/nodejs/CHANGELOG.md:22]: Hm but maybe we could also just run without bumping the version, there was no release to gh, but not sure if the workflow fails if it was already published to npm

Comment (thibault-martinez) [reply bindings/nodejs/CHANGELOG.md:22]: It's fine, let's do 1.1.3

**Review by thibault-martinez — APPROVED** [review]

**Review by Alex6323 — APPROVED** [review]

File: bindings/nodejs/package.json [hunk 2]
```diff
@@ -43,7 +43,7 @@
         "eslint-config-prettier": "^8.5.0",
         "jest": "^29.4.2",
         "jest-matcher-utils": "^29.5.0",
-        "prebuild": "^3.0.3",
+        "prebuild": "^12.1.0",
```
Comment (Alex6323) [inline bindings/nodejs/package.json:46]: that's quite a bump

**Pull request merged**